run-ci-tests.sh

```diff
@@ -14,8 +14,7 @@ case $TEST_TYPE in
 
 "Reducer integration tests")
   echo "Running reducer integration tests"
-  ./run src/examples/zkapps/reducer/actions-as-merkle-list.ts --bundle
-  ./run src/examples/zkapps/reducer/actions-as-merkle-list-iterator.ts --bundle
+  ./run src/examples/zkapps/reducer/run.ts --bundle
```

Comment:
Change here is because previously these files would run a local test as they were imported.  I wrapped the tests in an export and call them explicitly in this run file.  This allows the run-live file to import the same smart contracts without too many excess diffs.